src/Features/Core/Portable/CodeLens/CodeLensReferencesService.cs

```diff
@@ -246,7 +246,7 @@ private static async Task<ReferenceMethodDescriptor> TryGetMethodDescriptorAsync
                 return null;
             }
 
-            var document = solution.GetDocument(doc.Id);
+            var document = await solution.GetDocumentAsync(doc.Id, includeSourceGenerated: true, cancellationToken).ConfigureAwait(false);
```

Comment:
i honestly don't understand the point of the before/after line in teh first place.  Right above this, we already found teh 'doc', so why not just use that?

Reply:
@CyrusNajmabadi Honestly, me too. Let me delete it and see if there are any failures that explains this.

Reply:
I'm guessing this got refactored over the years and ended up in this form without anybody realizing it.

Reply:
This goes back to https://github.com/dotnet/roslyn/commit/b83e57778fda657b1b1b6f7ddc5c92cecaac3036